### PR TITLE
Custom partitioning for Ubuntu

### PR DESCRIPTION
**Pull request by sestegra**
In order to have a good experience with our community, we recommend that you read the [contributing guidelines](https://github.com/rainpole/packer-vsphere/blob/main/CONTRIBUTING.md) for making a pull request.

**Summary of Pull Request**

Apply CIS partitioning on Ubuntu

**Type of Pull Request**

- [ ] This is a bug fix.
- [X] This is an enhancement or feature.
- [ ] This is a code style / formatting update.
- [ ] This is a documentation update.
- [ ] This is a refactoring update.
- [ ] This is something else.
      Please describe:

**Context of the Pull Request***

Update autoinstall configuration file to apply custom partitioning

**Related to Existing Issues**

Issue Number: N/A

**Test and Documentation Coverage**

- [x] Tests have been completed (for bug fixes / features).
- [ ] Documentation has been added / updated (for bug fixes / features).

**Breaking Changes?**

- [ ] Yes, there are breaking changes.
- [X] No, there are no breaking changes.

### CODE REVIEW

**Comment by sestegra** [comment]
Ubuntu 18.04 is using `bios`

**Comment by sestegra** [comment]
Ready to merge

File: .gitignore [hunk 1]
```diff
@@ -31,6 +31,7 @@ terraform.rc
 ## Ignore default config directory.
 config/
 ## Ignore certificates files.
+**/*.cer
```
Comment (tenthirtyam) [inline .gitignore:34]: Good catch with my recent change. :)

**Review by tenthirtyam — APPROVED** [review]
✅ Reviewed.
✅ E2E Tested.
🚀 Approved!

**Pull request merged**